contrib/test_jenkins.sh

```diff
@@ -235,6 +235,19 @@ get_ifaddr() {
 	echo $(ip addr show ${iface} | awk '/inet /{print $2}' | awk -F '/' '{print $1}')
 }
 
+#
+# Check IB devices on state
+#
+#
+check_machine() {
+	device_list=$(ibv_devinfo -l | tail -n +2 | sed -e 's/^[ \t]*//' | head -n -1)
+	for ibdev in $device_list
+	do
+		port=1
```

Comment:
`num_ports=$(ibv_devinfo -d $ibdev| awk '/phys_port_cnt:/ {print $2}')`
`for port in $(seq 1 $num_ports)`